### DISCUSSION TITLE
Edge feature in GCNConv

### DISCUSSION THREAD

**Discussion by shrimonmuke0202** [pyg-team/pytorch_geometric#4257] [Q&A]
Dear all,
I want the feed edge feature in the GCNConv, and my shape of the edge feature is (# edges, 2). But in the documentation of GCNConv, the shape of the edge feature is (# of edges). Please help me with this issue.

**Comment by rusty1s** [comment]
The `GCNConv` only supports one-dimensional edge weights. If you want to pass in multi-dimensional edge features into your GNN, you can pick one of the GNN layers that supports them (see [here](https://pytorch-geometric.readthedocs.io/en/latest/notes/cheatsheet.html) and look out for GNN layers that support `edge_attr`) or write your own variant of GCN with support for it. Let me know if this is helpful to you :)
